tools/psi-visualizer/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pandas
```

Comment:
please add version to requirements from pip show, that's going to guarantee you stable behaviour, because rn it'll install latest version and that might break at some point (and also different machines could assume different default version)

Reply:
will do